CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+     Keep each description line below 72 characters.
+
+
+Managing of Issues and Pull Requests
+------------------------------------
+
+The cocotb project makes use of GitHub labels attached to issues and pull requests to structure the development process.
+Each issue and pull request can have multiple labels assigned.
+
+The `type` labels define the type of issue or PR:
+- `type:bug`: a bug in existing functionality
+- `type:feature`: new functionality
+- `type:question`: a support question
+
+The `status` labels give a quick impression of the current status of the issue or PR:
+- `status:worksforme`: the issue it not reproducible, or intended behavior (i.e. not a bug)
+- `status:on-hold`: further progress is blocked by a dependency, e.g. other code which must be commited first.
+- `status:needinfo`: feedback from someone is required. The issue/PR text gives more details.
+- `status:duplicate`: the same issue is already being handled in another issue/PR.
+
+Specificially for the use in pull requests the following additional status labels are defined:
+- `status:review-needed`: this PR needs at least one review
+- `stauts:changes-requested`: changes are requested to the code
+- `status:ready-for-merge`: this PR is ready (according to the [Patch Requirements](#patch-requirements)) to be merged
+
+The `category` labels help maintainers to filter issues which are relevant to their area of expertise:
+- `category:windows`: Microsoft Windows-specific issues
+- `category:simulators`: simulator support, including VPI/GPI/etc.
+- `category:packaging`: issues related to (PyPi) packaging, etc.
+- `category:docs`: documentation issues and fixes
+
+To help new contributors find a good issue to work on one more label is used (following [GitHub standard practices](#https://help.github.com/articles/helping-new-contributors-find-your-project-with-labels/)):
+- `good first issue`: this issue is a good starting point for new contributors.
+  The issue should give an actionable description of what to do to complete this task, along with contact information of a mentor for this task.
+
+cocotb explicitly uses no priority labels, as experience indicates that they provide little value.
+
+Issues and pull requests which are invalid, or where feedback is lacking for four weeks, should be closed.
+
+Cocotb Releases
+---------------
+
+cocotb aims to keep the `master` branch always in a releasable state.
+At least four times a year an official release should be created.
+It is the job of the maintainers to find a suitable time for a release, to communicate it to the community, and to coordinate it.
+
+
+Maintainers
+-----------
+
+Cocotb uses a shared maintainer model.
+Most maintainers are experts in part of the cocotb codebase, and are primarily responsible for reviews in this area.
+
+- Luke Darnell (@lukedarnell)
+- Julius Baxter (@juliusbaxter)
```

Comment:
You appear to want to sort by last name, in which case, Julius should move one position up.

Reply:
good catch, fixed as well.